packages/search-engine/__tests__/search-engine.test.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const searchEngine = require('..');
+
+describe('@booking-services/search-engine', () => {
```

Comment:
When you work with test, let’s have in mind that we need to add tests here using the  `given-when-then` approach.

Example:
`test('Given a <context> when an <action> then a <result>')`